metapool/sample_sheet.py

```diff
@@ -944,6 +946,54 @@ def __init__(self, path=None):
         }
 
 
+class MetatranscriptomicSampleSheetv10(KLSampleSheet):
+    _HEADER = {
+        'IEMFileVersion': '4',
+        'SheetType': _STANDARD_METAT_SHEET_TYPE,
+        'SheetVersion': '10',
+        'Investigator Name': 'Knight',
+        'Experiment Name': 'RKL_experiment',
+        'Date': None,
+        'Workflow': 'GenerateFASTQ',
+        'Application': 'FASTQ Only',
+        'Assay': _METATRANSCRIPTOMIC,
+        'Description': '',
+        'Chemistry': 'Default',
+    }
+
+    # MaskShortReads and OverrideCycles are present
+    # "Well_description" column contains concatenated information
+    # (Sample_Plate + Sample_Name + well_id_384) vs. just the sample_name
+    # in previous iterations.
+
+    data_columns = ['Sample_ID', 'Sample_Name', 'Sample_Plate', 'well_id_384',
+                    'I7_Index_ID', 'index', 'I5_Index_ID', 'index2',
+                    'Sample_Project', 'total_rna_concentration_ng_ul',
+                    'vol_extracted_elution_ul', 'Well_description']
+
+    _BIOINFORMATICS_COLUMNS = frozenset({'Sample_Project', 'QiitaID',
+                                         'BarcodesAreRC', 'ForwardAdapter',
+                                         'ReverseAdapter', 'HumanFiltering',
+                                         'contains_replicates',
+                                         'library_construction_protocol',
+                                         'experiment_design_description',
+                                         'contains_replicates'})
+
+    def __init__(self, path=None):
+        super().__init__(path=path)
+        self.remapper = {
+            'sample sheet Sample_ID': 'Sample_ID',
+            'Sample': 'Sample_Name',
+            'Project Plate': 'Sample_Plate',
+            'Well': 'well_id_384',
+            'i7 name': 'I7_Index_ID',
+            'i7 sequence': 'index',
+            'i5 name': 'I5_Index_ID',
+            'i5 sequence': 'index2',
+            'Project Name': 'Sample_Project',
+        }
```

Comment:
for absolute quant sample_sheets we included 1 dictionary key:value pair for each data_column that we expected in the samplesheet. Thus, do we need to remap `'vol_extracted_elution_ul':'vol_extracted_elution_ul'` and `'Sample RNA Concentration':'total_rna_concentration_ng_ul' `? In this case  'Sample RNA Concentration' will be the front facing column header (notebook interface) while 'total_rna_concentration_ng_ul' is the samplesheet and qiita header. Alternatively, I can name the 'Sample RNA Concentration' column with 'total_rna_concentration_ng_ul' in the notebook. Will the make_sample_sheet() function find the column defined in `data_columns` in the plate_df objects that gets passed as an argument and populate the associated data to the samplesheet?

Reply:
To be honest, I don't have enough information to answer that last part. The first part is easy enough to add into the dictionary for that class, if we need it. I propose we approve this PR and test it to see if we can resolve your two questions.